03_building_and_packaging/spack_exercise.md

```diff
@@ -18,56 +18,29 @@ The [code repository](https://github.com/Simulation-Software-Engineering/spack-e
 - `v0.2.0` depends on Boost (at least version `1.65.1`)
 - `v0.3.0` depends on Boost (at least version `1.65.1`) and `yaml-cpp` (at least version `0.7.0`)
 
-Note that we require `yaml-cpp` in version `0.7.0` again. This package will be automatically installed via Spack if you specify the dependency correctly in you package. You should **not* install `yaml-cpp` manually in this exercise.
+**Note**: We require `yaml-cpp` in version `0.7.0` again. This package will be automatically installed via Spack if you specify the dependency correctly in you package. You should **not* install `yaml-cpp` manually in this exercise.
 
-Create a Spack package for all releases of the given code and make sure that the dependencies are specified appropriately. Make sure you also add yourself as maintainer to your package. Your final package should not contain any `FIXME` parts anymore.
+Create a Spack package for all releases of the given code and make sure that the dependencies are specified appropriately. Make sure you also add yourself as maintainer to your package. Your final package should not contain any `FIXME` parts.
 
 ### Development/Packaging Environment
 
-Please do all the development inside a Docker container using the image [`ajaust/spack-package-exercise`](https://hub.docker.com/r/ajaust/spack-package-exercise). This should help to avoid problems with setting up Spack and minimized the number of external packages that Spack needs to build.
-
-You can find the recipe of the image inside the `docker/` directory of [the example code on GitHub](https://github.com/Simulation-Software-Engineering/spack-exercise). The image itself is based on Ubuntu 20.04 and has the Boost dependency preinstalled. Additionally, Spack has been set up such that you
-
- and two editors `vim` and `nano` preinstalled. If you want to install further software in your container you are free to do so.
+Please do all the development inside a Docker container. The Docker container is based on the image built from the recipe provided in the [exercise repository](https://github.com/Simulation-Software-Engineering/spack-exercise). You can find the recipe of the image inside the `docker/` directory of [the example code on GitHub](https://github.com/Simulation-Software-Engineering/spack-exercise). The image itself is based on Ubuntu 20.04 and has the Boost dependency preinstalled. Additionally, Spack has been set up in the recipe. Two editors `vim` and `nano` preinstalled. If you want to install further software in your container you are free to do so.
```

Comment:
Any specific reason why 20.04?

Reply:
With Ubuntu 22.04 there is a compiler conflict for `yaml-cpp` if used via Spack and without any additional compiler installations. I did not investigate this further in the interest of time, but will do so post-lecture.